src/DatePicker.elm

```diff
@@ -412,9 +419,24 @@ datePicker { today, currentMonth, currentDates, pickedDate, settings } =
         onPicker ev =
             Json.succeed
                 >> onWithOptions ev
-                    { preventDefault = True
+                    { preventDefault = False
```

Comment:
Why this change? `onPicker` seems to be attached to the whole picker, so presumably it has some effect outside the year picker itself?